### PR TITLE
Release

### PR DESCRIPTION
**Pull request by NeonzGamingYT**
just added release thing
so they can find the highlighted things

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -49,7 +50,8 @@ If you find problems when you run MCinaBox, you can check [Q&A](./Q&A.md) first.
 ### Build
 - Build via Android Studio.
 
-#### You can also get released apk from [here](https://github.com/AOF-Dev/MCinaBox/releases).
+## Release
+- You can also get released apk from [here](https://github.com/AOF-Dev/MCinaBox/releases).
```
Comment (TSaltedfishKing) [inline docs/README.md:54]: User can find it

Comment (TSaltedfishKing) [reply docs/README.md:54]: so it's unless

Comment (NeonzGamingYT) [reply docs/README.md:54]: but its cool

Comment (NeonzGamingYT) [reply docs/README.md:54]: please this is my very first thing to try

**Pull request merged**